perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/FacilitiesManagerMethod.java

```diff
@@ -523,13 +523,19 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 	 * Parameter description is optional.
 	 * Other parameters are ignored.
 	 * @param facility Facility JSON object
+	 * @throw ConsentHubExistsException if consent hub with facility name exists
+	 * @throw FacilityExistsException if facility already exists
+	 * @throw PrivilegeException for insufficient rights
```

Comment:
PrivilegeExceptions are added automatically, I think.

Reply:
good to know, fixed